README.md

```diff
@@ -17,83 +17,318 @@ Main characteristics:
+});
+```
+
+### derived vs computed
+
+While derived and computed may appear similar, they exhibit distinct characteristics that can significantly impact effectiveness based on use-cases:
+
+- **Declaration of Dependencies:**
+  - `computed`: No explicit declaration of store dependencies is required, providing more flexibility in code composition.
+  - `derived`: Requires explicit declaration of dependencies.
+
+- **Performance:**
+  - `computed`: Better performance by re-running the function only based on changes in the stores involved in the last run.
+  - `derived`: Re-run the function each time a dependent store changes.
+
+- **Asynchronous State:**
+  - `computed`: Unable to manage asynchronous state.
+  - `derived`: Can handle asynchronous state with the `set` method.
+
+- **Skipping Value Emission:**
+  - `computed`: Does not provide a mechanism to skip emitting values.
+  - `derived`: Allows skipping the emission of values by choosing not to call the provided `set` method.
+
+- **Setup and Teardown:**
+  - `computed`: Lacks explicit setup and teardown methods.
+  - `derived`: Supports setup and teardown methods, allowing actions such as adding or removing DOM listeners.
+
+While `computed` feels more intuitive in many use-cases, `derived` excels in scenarios where `computed` falls short, particularly in managing asynchronous state and providing more granular control over value emissions.
+
+Carefully choosing between them based on specific requirements enhances the effectiveness of state management in your application.
+
+### Getting the value
+
+There are three ways for getting the value of a store:
+
+```typescript
+import {writable, get} from "@amadeus-it-group/tansu";
+
+const count$ = writable(1);
+const unsubscribe = count$.subscribe((count) => {
+  // Will be called with the updated value, synchronously first, then each time count$ changes
+  // must be unsubscribe to end the future calls.
+  console.log(count);
+});
+
+// A store is also a function that you can call to get the instant value.
+console.log(count$());
+
+// Equivalent to
+console.log(get(count$));
+```
+
+> [!NOTE]
+> Getting the instant value implies the subscription and unsubription on the store:
+>   - It can be important to know in case of setup/teardown functions.
+>   - In the same scope, prefer to store the value once in a local variable instead of calling `store$()` several times.
+>
+> `get` can work with Class variant of a store,
+> When called inside a reactive context (i.e. inside a computed), getting the value serves to know and "listen" the dependent stores.
+
+
+### batch
+
+[api documentation](https://amadeusitgroup.github.io/tansu/functions/batch.html)
+
+Contrary to other libraries like Angular with signals or Svelte with runes, where the callback of a subscription is executed asynchronously (with `useEffect`), we have maintained the constraint of synchronicity between the store changes and their subscriptions in Tansu.
```

Comment:
`useEffect` is neither in Angular nor in Svelte but in React (and it is not linked to a signal).
You probably meant `effect` in Angular or `$effect` in Svelte.

Reply:
You're right.